.github/actions/f3d-superbuild/action.yml

```diff
@@ -57,6 +56,7 @@ runs:
     working-directory: ${{github.workspace}}/build
     shell: bash
     run: |
+      cmake ./
```

Comment:
why is it necessary?

Reply:
See the description of the PR.

the Superbuild generate a version number in f3d-version.cmake at configuration time, however, when building from git, the default value is used to generate (1.3.1). When we rerun it after the build, the superbuild generates it again using git information as it is available.

I will document that in the final MR updating the readme.

For more info, ptal: https://gitlab.kitware.com/paraview/common-superbuild/-/blob/master/cmake/SuperbuildVersionMacros.cmake#L207